doc/developer-guides/sw_design_guidelines.rst

```diff
@@ -243,28 +226,10 @@ Here is the sample codes for error checking before calling ``vcpu_from_vid``:
 Module Level
 ============
 
-Fulfillment to FuSa Standards
------------------------------
-
-The fulfillment matrix of error detection and error handling on a module level
-to FuSa Standards [IEC_61508-7]_ in ACRN hypervisor is shown in
-:numref:`fulfillment_matrix_module_level` below.
-
-.. table:: Fulfillment to FuSa Standards on Module Level
-   :align: center
-   :widths: auto
-   :name: fulfillment_matrix_module_level
-
-   +-----------------------+---------------+----------+---------------------------------------------------+
-   | Technique/Measure     | Ref in        | SIL3     | Company Solution                                  |
-   |                       | IEC 61508-7   |          |                                                   |
-   +=======================+===============+==========+===================================================+
-   | Design and coding     | C.2.6         | HR       | This technique will be used for internal functions|
-   | standards             | Table B.1     |          | of the hypervisor. Plan to use data verification, |
-   |                       |               |          | with explicit specification of pre-conditions and |
-   |                       |               |          | post-conditions for functions.                    |
-   +-----------------------+---------------+----------+---------------------------------------------------+
+Related Requirements of FuSa Standards
+--------------------------------------
```

Comment:
Similarly, can we leave this in the document:
```
Design and coding standards are used for error detection
and error handling as recommended by FuSa Standards, and
used for internal functions of the hypervisor. The ACRN hypervisor
uses data verification, with explicitly documented specification of
pre-conditions and post-conditions for functions.
```

Reply:
Hi David, thanks for your comments, error detection and error handling is not described by IEC 61508, it needs to more effort that make complete and clear fulfillment matrix in the safety plan about error detection and error handling. Can we just keep reference for open source in this design guidelines document? BTW, this simple description maybe confuse some FuSa experts. We will make complete fulfillment matrix to cover this in the safety plan.

Reply:
@hwang37 Do you have any comments about this? thanks!

Reply:
Same question here.  The IEC documents won't say "The ACRN hypervisor
uses data verification, with the explicitly documented specification of
pre-conditions and post-conditions for functions."  So would we want to write this here?

Reply:
Just plan to move fulfillment table into safety plan, if safety plan is ready, we maybe refer safety plan directly. I will update related description which is not in the IEC standard. Thanks!